app/src/ui/tutorial/tutorial-panel.tsx

```diff
@@ -64,7 +64,16 @@ export class TutorialPanel extends React.Component<
   }
 
   private openPullRequest = () => {
-    this.props.dispatcher.createPullRequest(this.props.repository)
+    // This will cause the tutorial pull request step to close first.
+    this.props.dispatcher.markPullRequestTutorialStepAsComplete(
+      this.props.repository
+    )
+
+    // wait for the tutorial step to close before opening the PR, so that the
+    // focusing of the "You're Done!" header is not interupted.
+    setTimeout(() => {
+      this.props.dispatcher.createPullRequest(this.props.repository)
+    }, 500)
```

Comment:
Not the happiest about this, but couldn't think of way to ensure it was sequential. The issue I am running into is that the details component in the tutorial would grab the next focus and then the app would switch to browser before setting focus in the `Done` component. Thus, when the user returned to the app it would still be on the tutorial pane.